### PR TITLE
close #2 documentation update

### LINKED ISSUE #2: Flesh out README
# Motivation

To better communicate and establish what `eigenflow` is intended and good for we need to add more documentation which provides some example usecases and highlights some comparables.
# Input

Current [README.md](https://github.com/ypg-data/eigenflow/blob/master/README.md)
# Output

Updated [README.md](https://github.com/ypg-data/eigenflow/blob/master/README.md) 
# Test
- [x] Clear specification of the problem `eigenflow` is designed to handle
- ~~[ ] Comparison and contrast between `eigenflow` and at least two other similar things~~
- [x] Clear specification of what `eigenflow` is not intended for

### PR DESCRIPTION
**Pull request by dmitri-carpov**

### CODE REVIEW

**Comment by jonas** [comment]
A link to the initial version and its comments: https://github.com/dmitri-carpov/eigenflow/commit/9263000a2371b579b900621b7a4ab1034f388041

**Comment by jonas** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -35,58 +32,48 @@ val sendReport = SendReport { newReportFile =>
...
+### What it is good for
+
+`Eigenflow` was created for managing periodic long-running ETL processes with automatic recovery of failures.
+When stages performance is important and there is a need to collect statistics and monitor processes.
+
 
-* The system will automatically do "checkpoints" on every stage switch, thus in case of failure it restarts by running the
-failed stage again.
-* It can automatically catch-up by processing missing days, and many more - see [complete list of features](#main-features).
+### What it may not be a good for
```
Comment (suhailshergill) [inline README.md]: `s/a good/good/`

Comment (suhailshergill) [inline README.md]: rest looks good, +1 after minor tweak above

**Pull request merged**